### DISCUSSION TITLE
Graphenos issue

### DISCUSSION THREAD

**Discussion by Grkrz** [tanujnotes/Olauncher#192] [Q&A]
Hej
The app works slow on graphenos
Sometimes I have to tap on the app twice before it opens. There is also problem with animations. Thank you.

**Comment by tcarrio** [comment]
> Hej
> The app works slow on graphenos
> Sometimes I have to tap on the app twice before it opens. There is also problem with animations. Thank you.

Currently running OLauncher on GrapheneOS on a Pixel 6a, I have not seen this issue.

You could provide more hardware info to perhaps help determine the cause of your issue

**Reply by Grkrz** [reply]
Hej 
The problem is gone.
Thanks for your feedback
